content/docs/users/_index.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "Users"
+weight = 100
+template = "docs/index.html"
+[extra]
+emoji = "🧑"
+tile = "I want to start chatting using Matrix"
+updated = "2022-10-18T16:00:00Z"
++++
+
+## Matrix for Instant Messaging
+
+Matrix works a little like email, but instantaneous and secure:
+- You need to register an account at a provider
+- Whatever your provider is, you can talk to people using other providers
+- In the same way you can use Outlook or Thunderbird with the same email
+  account, you can use different Matrix apps for the same Matrix account.
+
+Several apps exist, but we're going to go with Element for the sake of
+simplicity, as it's among the most advanced apps on the market.
+
+Once you are more comfortable with the basics and if you want to use another
+app, head to the [clients](/ecosystem/clients) section of this website.
+
+## Creating a Matrix account
+
+You can use any provider you want. The experts can even set-up their own
+provider, but it's not a requirement at all.
+
+The Matrix.org Foundation maintains a public provider everybody can register
```

Comment:
Should be "public server", right? If we want to use "provide(r)", we could replace "maintains"?

Reply:
Good catch indeed! How about "The Matrix.org Foundation is a public provider everyone can register […]"?

Reply:
Sure.